File: CodingSamples/Java/Language-Runtime/PrimitiveTypeTest/demo/Investment.java
class Investment {

    //a method declared with 'public' modifier is visible outside of its declaring class
    //a method declared with 'static' modified can be called directly on the declaring class
    public static double futureValue(double installment, int duration, boolean risk) {
        double i = risk ? 0.08 : 0.06;
        return (installment / i) * (Math.pow(1 + i, duration) - 1);
    }
}

File: CodingSamples/DotNet/Language-Runtime/PrimitiveTypeTest/DemoApp/Investment.cs
class Investment
{
    //a method can declare optional parameters at the end of its parameter-list
    //along with their default argument values
    public static double FutureValue(double installment, int duration, bool risk = false)
    {
        double i = risk ? 0.08 : 0.06;
        return (installment / i) * (System.Math.Pow(1 + i, duration) - 1);
    }
}
